interview-questions.md

```diff
@@ -8,62 +8,66 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is the difference between a parameter and an argument?
 
-  Your answer:
-
-  Researched answer:
-
+  Your answer: A parameter is defined by a function and is found within parentheses after the function name. An argument affects change within the function. 
 
+  Researched answer: Where the function is invoked, arguments are found in parentheses after the function name, and serve to pass content to the function. For every argument, there is an equal number of paremeters. Arguments can be different data types such as an array of numbers, or strings. 
 
 2. The JavaScript built in method .map() takes predefined parameters. What are they? Which are required and which are optional?
 
-  Your answer:
-
-  Researched answer:
+  Your answer: The predefined paremeters taken by the .map method are values that lie within an array. I do not recall which parameters are required and which are optional, but I will gladly do more research on this method and follow up with my findings. 
```

Comment:
Great answer for an interview setting!  Share what you do know and show that you are curious on what you don't know and will research it.  Well done!